src/components/SearchBar.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="input-icons">
```

Comment:
there's only one icon right? Why is this pluralized? Also isn't this just the search-container?